.pre-commit-config.yaml

```
@@ -108,3 +108,9 @@ repos:
     language: system
     verbose: true
     pass_filenames: false
+  - id: check-filenames
+    name: Check for spaces in all filenames
+    entry: bash -c 'git ls-files | grep " " && echo "Filenames should not contain spaces!" && exit 1 || exit 0'
```

Is it nicer if we split it into multiple lines like the signoff-commit line?
```suggestion
    entry: bash 
    args: 
      - -c 
      - 'git ls-files | grep " " && echo "Filenames should not contain spaces!" && exit 1 || exit 0'
```

---

Thanks for the suggestion, followed up in https://github.com/vllm-project/vllm/pull/12880